contributors/join-us/localization.md

```diff
@@ -108,6 +108,7 @@ If you're interested in contributing to the process, please join the [Mattermost
 
 | Language | Official Reviewer\(s\) | Maintainers |
 | :--- | :--- | :--- |
+| Bulgarian | [Nikolai Zahariev \(nikolaiz\)](https://github.com/mattermost/mattermost-handbook/tree/55178956e5d33dde426723e7eb3897c9fcacc52e/contributors/contributors/translate.mattermost.com/user/nikolaiz/README.md) | |
```

Comment:
Is this intended to link to https://translate.mattermost.com/user/nikolaiz/?

Reply:
Yes. I'll get that corrected.